docs/kata-orchestrator-reference-skills/skills/kata-doctor/scripts/check-roadmap-format.sh

```
@@ -0,0 +1,43 @@
+
+# Canonical format requires ALL of:
+# 1. "## Milestones" section (overview list with ✅/🔄/○ symbols)
+# 2. "## Current Milestone:" heading (either "v[X.Y] [Name]" or "None")
+# 3. Either "## Completed Milestones" section OR no completed work yet
+
+HAS_MILESTONES=$(grep -E "^## Milestones" "$ROADMAP" 2>/dev/null)
+HAS_CURRENT_MILESTONE=$(grep -E "^## Current Milestone:" "$ROADMAP" 2>/dev/null)
+
+# Check for completed milestones section OR verify it's a new project
+# (new projects won't have completed milestones yet, which is OK)
+HAS_COMPLETED_SECTION=$(grep -E "^## Completed Milestones" "$ROADMAP" 2>/dev/null)
```

Shellcheck (0.11.0)

[warning] HAS_COMPLETED_SECTION appears unused. Verify use (or export if used externally).

(SC2034)